includes/class-sensei-lesson.php

```diff
@@ -2272,61 +2272,6 @@ public function add_column_data( $column_name, $id ) {
 		} // End Switch Statement
 	} // End add_column_data()
 
-	/**
-	 * Add a course from the lesson page.
-	 *
-	 * @access public
-	 * @deprecated 2.2.0
```

Comment:
I'm wondering if we should remove these 2 methods deprecated in `2.2.0` now or wait for until the `4.0`. I think it would fit on `User-Facing Functionality` of our policy.
WDYT @alexsanford?

Reply:
Yes, I think this should be removed in `4.0`, since it is `public` and the comment doesn't set `@access private`.

Reply:
Good catch! Added that to #2407 instead.